pkg/hintrunner/utils/secp_utils.go

```diff
@@ -6,6 +6,9 @@ import (
 	"math/big"
 
 	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+	// "github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
```

Comment:
what are these for?

Reply:
I think this was also coming from the draft PR, not sure why 🤔